### PR TITLE
Delete selection area from RootScreen.

### LINKED ISSUE #235: セッションタブ表示中に、ホームタブのテキストが選択できてしまう
**Describe the bug**
セッションタブにて画面を長押しすると、ホーム画面にあるテキストが「コピー　すべて選択」できるようになっている

**To Reproduce**
1. セッションタブを開く
2. 基調講演の枠と、Add Material touch ripplesの枠の間のあたりを長押しする
3. 「コピー　すべて選択」のコンテキストメニューが表示される
4. コピーすると、ホームタブにあるテキスト（「FlutterKaigi」など）がコピーされる

**Expected behavior**
ホームタブ以外を開いているときは、ホームタブのテキストがコピーできないようになる

**Screenshots**

image

**Smartphone (please complete the following information):**
 - Device: Android

### PR DESCRIPTION
**Pull request by Kurogoma4D**
## Description

- Delete selection area from RootScreen.
- Put selection area on HomeScreen while on web.

Fixes #235

## Type of change

Please delete options that are not relevant.

- [ ] New feature
- [x] Bug fix
- [ ] Refactor
- [ ] Style
- [ ] Documentation
- [ ] CI/CD
- [ ] Other

## How Has This Been Tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.

## Checklist

- [x] My code follows the style guidelines of this project.
- [x] My code has been formatted with `flutter format lib`.
- [x] My code has been fixed with `dart fix --apply lib`.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: lib/ui/screen/home.dart [hunk 1]
```diff
@@ -139,5 +140,7 @@ class HomePage extends ConsumerWidget {
         ],
       ),
     );
+
+    return kIsWeb ? SelectionArea(child: contents) : contents;
```
[the remark this answers is not in the document]
Comment (Kurogoma4D) [reply lib/ui/screen/home.dart:144]: 早期リターンにしても、

```dart
if (!kIsWeb) {
  return contents;
}

return SelectionArea(
  child: contents,
);
```

で可読性も処理効率も変わらないかなと思いましたがどうでしょう？

[the remark this answers is not in the document]
Comment (koji-1009) [reply lib/ui/screen/home.dart:144]:
```dart
if (kIsWeb) {
  /// Allow text selection only in the browser
  return SelectionArea(
    child: contents,
  );  
}

return contents;
```

こんなイメージでした！
意図のコメントには、if文の方がいいかなと。

[the remark this answers is not in the document]
Comment (Kurogoma4D) [reply lib/ui/screen/home.dart:144]: 承知です〜
doc commentはこの場面では不適切だと思うので、通常のコメントにして修正します！

**Review by koji-1009 — APPROVED** [review]
LGTM!

File: lib/ui/screen/home.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (koji-1009) [reply lib/ui/screen/home.dart]: ありがとうございます！！！

**Pull request merged**